examples/calling-apis/langchain-examples/src/auth0/auth0_ai.py

```diff
@@ -0,0 +1,22 @@
+from dotenv import load_dotenv
+from auth0_ai_langchain.auth0_ai import Auth0AI
+
+load_dotenv()
+
+auth0_ai = Auth0AI()
+
+
+with_calendar_free_busy_access = auth0_ai.with_federated_connection(
+    connection="google-oauth2",
+    scopes=["https://www.googleapis.com/auth/calendar.freebusy"]
+)
+
+with_slack_access = auth0_ai.with_federated_connection(
+    connection="sign-in-with-slack",
+    scopes=["channels:read", "chat:write"]
+)
+
+with_github_access = auth0_ai.with_federated_connection(
+    connection="github",
+    scopes=["repo"],
+)
```

Comment:
please remember to update the `examples/calling-apis/langchain-examples/README.md` (for example, the Prerequisites section)